paths/electron/starting-with-electron/08-add-to-js.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Add JavaScript
+permalink: /electron/starting-with-electron/add-to-javascript
+next-page: /electron/starting-with-electron/celebrate
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  Finally, let's add the fun part: the Javascript!
+
+  1. Copy the Javascript from Codepen into the `src/script.js` file.
+  1. Spin up your app, and see how it works.
+  1. Save and commit your changes.
```

Comment:
If image path proves to add complexity and we need to troubleshoot with a new Codepen, @nathos and I used https://codepen.io/jackrugile/pen/aCzHs . This has its own library dependencies, but was simple enough to feel reliable. 🤷‍♂️ No strong opinions either way!